### PR TITLE
IDK what i'm doing

### PR DESCRIPTION
**Pull request by Pinduca23**
Yes

### CODE REVIEW

File: PassPy/password_manager.py [hunk 1]
```diff
@@ -7,16 +7,14 @@
 
 def pass_size():
     # Gets desired password length
-    while True:
+    pass_length = int(input('\nRequired size of password: '))
+    while pass_length < 4:
```
Comment (plinioh) [inline PassPy/password_manager.py]:
```suggestion
    pass_length = 0
    while pass_length < 4:
```

File: PassPy/password_manager.py [hunk 2]
```diff
@@ -7,16 +7,14 @@
...
         try:
+            print('\nPassword must have at least 4 characters or words')
```
Comment (plinioh) [inline PassPy/password_manager.py]:
```suggestion

```

File: PassPy/password_manager.py [hunk 3]
```diff
@@ -7,16 +7,14 @@
...
             pass_length = int(input('\nRequired size of password: '))
```
Comment (plinioh) [inline PassPy/password_manager.py:13]:
```suggestion
            pass_length = int(input('\nRequired size of password: '))
            if pass_length < 4:
               print('\nPassword must have at least 4 characters or words')
```

**Review by plinioh — CHANGES_REQUESTED** [review]

**Comment by Pinduca23** [comment]
Commit after code review

**Pull request merged**